guides/common/modules/proc_configuring-capsule-default-certificate.adoc

```diff
@@ -56,8 +55,8 @@ root@_{smartproxy-example-com}_:/root/_{smartproxy-example-com}_-certs.tar
 
 . On {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the `{certs-generate}` command returns.
 +
-When network connections or ports to {Project} are not yet open, you can set the `--foreman-proxy-register-in-foreman` option to `false` to prevent {SmartProxy} from attempting to connect to {Project} and reporting errors.
-Run the installer again with this option set to `true` when the network and firewalls are correctly configured.
+When network connections or ports to {Project} are not yet open, registration of the {SmartProxy} will fail.
+Correctly configure the network and firewalls before rerunning the installer.
```

Comment:
This should be covered in Prerequisites. It may need a different wording.

Reply:
I was thinking about the same thing.